save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -229,11 +280,15 @@ class VulnerabilityService(
             status = VulnerabilityStatus.CREATED,
             userId = user.requiredId(),
             organization = organizationNew,
-            tags = vulnerabilityDto.tags.joinToString(DATABASE_DELIMITER),
         )
         val vulnerabilityNew = vulnerabilityRepository.saveAndFlush(vulnerability)
         val newName = "SOTV-${LocalDateTime.now().year}-${vulnerabilityNew.id}"
 
+        val existedTags = tagRepository.findByNameIn(vulnerabilityDto.tags)
```

Comment:
do we need this check? `spring-data-jpa` will just update the existing record, if it is the same
and in our case, since we didnt change anything, the record wont change, isnt it?

Reply:
The point is that primary key is `id`, not name. Thus, we do not save duplicates